src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -511,6 +473,60 @@ export default class MutationMapperTool extends React.Component<
         );
     }
 
+    protected referenceGenomeSelector() {
```

Comment:
should be @computed

also, did you mean to make this protected?  i would think private if anything

Reply:
@alisman yes `@computed` would be proper, just because other similar methods in this file have `protected` so I declared `protected`  as well. I just asked @onursumer about this, for this particular class we don't have class extending use-case, so I'll make it `@computed` for it